### PR TITLE
gh #40 Removed L1 tests for dsEnableDTCP(), dsIsDTCPEnabled(), dsSetActiveSource()

### PR DESCRIPTION
**Pull request by krishna-1809**

### CODE REVIEW

File: src/test_l1_dsVideoPort.c [hunk 1]
```diff
@@ -1101,125 +1101,6 @@ void test_l1_dsVideoPort_negative_dsIsVideoPortActive(void) {
 	UT_LOG("\n Out %s\n", __FUNCTION__); 
 }
 
-
-/**
- * @brief Positive Test Scenarios for dsEnableDTCP()
- * 
- * **Test Group ID:** Basic: 01@n
- * **Test Case ID:** 017@n
```
Comment (hari22yuva) [inline src/test_l1_dsVideoPort.c:1109]: Please update the test case id for other test cases 
please remove from the test suit

**Review by hari22yuva — CHANGES_REQUESTED** [review]
please address the comments

File: src/test_l1_dsVideoPort.c (hunk 1 above)
Comment (krishna-1809) [reply src/test_l1_dsVideoPort.c:1109]: updated

Comment (hari22yuva) [reply src/test_l1_dsVideoPort.c:1109]: Please update the other test case also according to the interface file i.e. what need return for sink and source device.

**Comment by Ulrond** [comment]
@srinivasgtl to review

**Review by ishthiyaqahmed — CHANGES_REQUESTED** [review]
Test case ID's should be same in description and in test cases .

/**
 * @brief Positive Test Scenarios for dsIsHDCPEnabled()
 * 
 * **Test Group ID:** Basic: 01@n
 * **Test Case ID:** 023@n

void test_l1_dsVideoPort_positive_dsIsHDCPEnabled(void) {
	gTestID = 23;
	gTestID = 19;

**Comment by hari22yuva** [comment]
changes are addressed in this PR 
https://github.com/rdkcentral/rdk-halif-test-device_settings/pull/90/

**Pull request closed without merging**